src/components/Quote.vue

```diff
@@ -30,7 +30,7 @@ components.
 		@click.prevent="handleQuoteClick">
 		<div class="quote__main">
 			<div class="quote__main__author">
-				<h6>{{ getDisplayName }}</h6>
+				<h6 role="presentation">{{ getDisplayName }}</h6>
```

Comment:
Same as the heading level comment above:
> Btw I’d bet that we are skipping some heading levels here like h4 and h5, which is semantically incorrect. We should use the level which makes sense for the markup, and then style it to fit how we want it to look like.

Reply:
indeed, missed that one. maybe for the quote we could make it h3 as it's one level deeper

Reply:
I’d assume it stays the same level though. A heading wouldn’t go one level down just cause it’s in a quote. So h3, as per the additional comment above?

Reply:
with the screen reader you use the up/down arrow keys in browse mode and it will read the current lines.
if you'd navigate over this one it will speak out "$authorName heading level three". it could cause confusion as the user would believe it's a new section on the same level

the "blockquote" bit is read out when you hit down arrow after the heading.
maybe if we move this heading inside the blockquote it will be more understandable, as the blockquote would get announced first and the heading as being inside

Reply:
I've changed it to the same level as author now